internal/actions/go.mod

```diff
@@ -0,0 +1,3 @@
+module cloud.google.com/go/internal/actions
```

Comment:
Why do we need a separate module for this?

Reply:
Technically no, but if I want to keep using newer io.FS apis, yes.